### DISCUSSION TITLE
[FEATURE REQ] action -> webhook only approved if status_code = 200

### DISCUSSION THREAD

**Discussion by zwaehnn** [autobrr/autobrr#1696] [Ideas]
**Is your feature request related to a problem? Please describe.**
When a filter uses a webhook as action and not as external command, it always get's approved. 

**Describe the solution you'd like**
HTTP status codes > 300 should be rejected.
Or a solution like the one used for the "external" webhook where a desired HTTP status (200) can be specified.

**Describe alternatives you've considered**
I could use an "external" webhook and a Test-action. But that way the release would only be added to the release-tab if the external-webhook returned the desired status_code and a "rejected" is not possible

**Comment by zze0s** [comment]
Hi! There's plans to make the actions a lot more powerful with features like this and having sub-actions trigger depending on status and ordering and so on. No ETA tho.

**Comment by zwaehnn** [comment]
That's great to hear!
I'm looking forward to it. Until then I will have to make do with the external webhook and rejected releases not showing up.

**Comment by chrisfosterelli** [comment]
I think it'd be excellent if you could specify a set of pairs of status codes and results, for e.g. 200 => Success, 204 => Rejected, Other => Error. The default values could be something sensible like 200 => Success, Other => Error
